pybamm/util.py

```diff
@@ -331,13 +331,11 @@ def install_jax(arguments=None):  # pragma: no cover
     """
     Install compatible versions of jax, jaxlib.
 
-    Command Line Interface:
-    -----------------------
-    >>> pybamm_install_jax
-
-    optional arguments:
-    -h, --help   show help message
-    -f, --force  force install compatible versions of jax and jaxlib
+    Command Line Interface::
+        $ pybamm_install_jax
+    |   optional arguments:
```

Comment:
Will render it properly -
```suggestion
    Command Line Interface::

        $ pybamm_install_jax

    |   optional arguments:
```